aptos-move/framework/move-stdlib/sources/hash.move

```diff
@@ -5,4 +5,9 @@
 module std::hash {
     native public fun sha2_256(data: vector<u8>): vector<u8>;
     native public fun sha3_256(data: vector<u8>): vector<u8>;
+
+    spec sha3_256(data: vector<u8>): vector<u8> {
+        aborts_if [abstract] false;
+        ensures [abstract] len(result) == 32;
+    }
```

Comment:
no need to be in a separate spec file if it's only a few lines?

Reply:
yeah, in general it would be nice to have a separate spec file but since we only specify one function, it is OK to have it in one file